### ISSUE TITLE
Unable to extract plural forms from FormattedMessage component

### ISSUE THREAD

**Issue by RopoMen** [evenchange4/react-intl.macro#38]
Result in JSON file does not contain `defaultMessage`, only id and description. `defaultMessage` can also be in multiline, see [example](https://github.com/yahoo/react-intl/blob/master/examples/hello-world/src/App.js)
```
<FormattedMessage
              id="app.fooText"
              defaultMessage={`Found {foo, number} {foo, plural, one {foo} other {foos}}`}
              description="Plural foos"
              values={{foo: 1}} />
```

**Comment by evenchange4** [comment]
@RopoMen Thanks for reporting this issue. It should be fixed in `v0.3.3`.

**Comment by RopoMen** [comment]
Hi,
I tested this also and plural form extraction worked nicely!
